entity-management-web/pom.xml

```diff
@@ -107,6 +108,23 @@
             <version>4.2.0</version>
             <scope>test</scope>
         </dependency>
+        
+        <!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.dataformat/jackson-dataformat-xml -->
+		<dependency>
+		    <groupId>com.fasterxml.jackson.dataformat</groupId>
+		    <artifactId>jackson-dataformat-xml</artifactId>
+		    <!-- version>2.11.4</version-->
+		</dependency>
+		
+		<!-- https://mvnrepository.com/artifact/junit/junit -->
+		<dependency>
```

Comment:
we have to use junit jupitter, see user sets tests for appropriate configurations

Reply:
Changed!